src/components/formatters/test/src/SmartFactoryTestHelper.cc

```diff
@@ -432,31 +432,19 @@ std::shared_ptr<ISchemaItem> CSmartFactoryTest::InitStructSchemaItem_Common_1(
                                 TSchemaItemParameter<size_t>(500),
                                 TSchemaItemParameter<std::string>());
 
-  // Struct member image.
-  //
-  // Image struct
-  std::shared_ptr<ISchemaItem> image_SchemaItem =
-      ProvideObjectSchemaItemForStruct(struct_schema_items,
```

Comment:
As I see, function `ProvideObjectSchemaItemForStruct` is used only in this place, so if you delete this call here, you also need to delete this this function as unused.